app/src/observables/webworker-messaging/pyodide.worker

```diff
@@ -0,0 +1,103 @@
+// import './webworker.tsx'
+
+// import loadWheels from '../../python/load-wheels.py';
+// import setupDirectories from '../../python/setup-directories.py';
+
+
+// // interface PyodideWorker extends Worker {
+// //   pyodide: any;
+// //   Module: any;
+// //   languagePluginUrl: string;
+// //   document: any;
+// // }
+
```

Comment:
This was the webworker code. Webworkers currently don't work very well with pyodidie, so I have left this here for future use, once some of the webworker pyodide bugs are fixed.